wagn/features/update_includers.feature

```diff
@@ -8,6 +9,7 @@ Feature: Updates for Children of watched cards
   Background:
     Given I am signed in as Joe User
     And I create Book card "Ulysses"
+    And Jobs are dispatched
```

Comment:
I had a lot more of those but I included it in the "should be notified" step. 
The "Jobs are dispatche"  is needed here because  "And I create Book card "Ulysses"" adds notifications to the job queue and they stay there until we check later for notifications but then we don't expect those notifications.  
We could get rid of this by dispatching jobs if we make somebody watch something  (what we do in the next step)